packages/e2e-test-utils/src/create-user.js

```diff
@@ -10,14 +10,25 @@ import { switchUserToAdmin } from './switch-user-to-admin';
 import { switchUserToTest } from './switch-user-to-test';
 import { visitAdminPage } from './visit-admin-page';
 
+/**
+ * @typedef {'Subscriber' | 'Contributor' | 'Author' | 'Editor' | 'Administrator'} Role
```

Comment:
I don't think we should restrict this to basic roles, as there is no validation happening and probably shouldn't. Someone could create a custom role to test something.

It's still a valuable information to keep though as a comment.